scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -222,7 +222,9 @@
                      (into [] (map #(into {} (select-keys % [:hostname :resources])) offers-this-pool)))
       offers-this-pool))
 
-  (restore-offers [this pool-name offers]))
+  (restore-offers [this pool-name offers])
+
+  (container-defaults [_] {}))
```

Comment:
Nitcpick: Should we add a TODO that the k8s container defaults will be implemented later?

Reply:
Added a comment saying that we currently don't support container defaults for k8s